dev/all_tests.yaml

```diff
@@ -151,7 +156,18 @@ tests:
     args:
       alias: [standard_azure_provider_params]
 
-# Add appengine_smoke_test
-#    need to add config support first
+  appengine_smoke_test:
+    requires:
+      configuration:
+        appengine_account_enabled: true
+      services: [clouddriver, front50, gate, orca]
+    api: gate
+    args:
+      alias: [standard_appengine_provider_params]
+      source_repo_url: http://github.com/ewiseblatt/scratch
```

Comment:
Should point to a repository containing an App Engine application. In the past I've used one of the public App Engine sample applications, e.g., `https://github.com/GoogleCloudPlatform/python-docs-samples.git`. `app_directory_root` points to where the application lives inside the repository.

If you use the repository I suggested, `app_directory_root` would be `appengine/standard/hello_world`.